packages/output/src/browser/output-widget.tsx

```diff
@@ -117,10 +117,16 @@ export class OutputWidget extends ReactWidget {
         };
 
         if (this.outputChannelManager.selectedChannel) {
-            for (const text of this.outputChannelManager.selectedChannel.getLines()) {
-                const lines = text.split(/[\n\r]+/);
+            for (const outputChannelLine of this.outputChannelManager.selectedChannel.getLines()) {
+                const lines = outputChannelLine.text.split(/[\n\r]+/);
+                let className;
+                if (outputChannelLine.severity === OutputChannelSeverity.Error) {
+                    className = 'theia-console-error';
```

Comment:
I'm not sure these are the correct classes to use as they are contributed by `console`.
It is possible to build an application without `console`, and in this scenario, coloring present in the output view will not work.

Reply:
I created new classes in output package styles